problemtools/verifyproblem.py

```diff
@@ -470,6 +470,16 @@ def check(self, args):
             if not f[:-4] + '.in' in infiles:
                 self.error("No matching input file for answer '%s'" % f)
 
+        # Check for filename characters which cause problems when judging; in
+        # particular, do not allow "=" in the filename as it causes problems
+        # in the way Kattis uses isolate. There are probably other characters
+        # we might allow, but we err on the side of safety.
```

Comment:
I don't think comments like this belong here.  My view is that problemtools is primarily supposed to be an implementation of the problem format standard, and implementation details of Kattis like this one should not be relevant.

Reply:
Thanks; when I was originally writing this I looked for a regex in the standard and did not find it, but @niemela pointed me to it. Fixed in acdcdb3.